crates/matrix-sdk-crypto/src/olm/utility.rs

```diff
@@ -75,11 +71,11 @@ impl Utility {
             signature.get(key_id.to_string()).ok_or(SignatureError::NoSignatureFound)?;
         let signature = signature.as_str().ok_or(SignatureError::NoSignatureFound)?;
 
-        let ret =
-            match self.inner.ed25519_verify(signing_key, &canonical_json, signature.to_owned()) {
-                Ok(_) => Ok(()),
-                Err(_) => Err(SignatureError::VerificationError),
-            };
+        let signature = vodozemac::Ed25519Signature::from_base64(signature).unwrap();
```

Comment:
I think that was remarked before already, but why don't we have `?` here again?

From the context it is not clear to me, why this might not just panic, if I just gave it a non base64-string here... Is there something I am missing? then please explain that in a `.expect()` (rather than an opaque `unwrap`)

Reply:
Hmm, can't remember a previous remark, nor can I find one.

This is just an error, we should use `?`. Fixed by https://github.com/matrix-org/matrix-rust-sdk/pull/494/commits/5a1e2d0a60f99aa5ff8843be3b32a645158e35de.